packages/aml-checklist/src/core/router.tsx

```diff
@@ -7,10 +7,32 @@ import PrivateRouteWrapper from './private-route-wrapper'
 
 export const history = createBrowserHistory()
 
-const Login = React.lazy(() => import('../components/pages/login'))
-const ChecklistDetail = React.lazy(() => import('../components/pages/checklist-detail'))
-const ClientSearch = React.lazy(() => import('../components/pages/client-search'))
-const Results = React.lazy(() => import('../components/pages/results'))
+export const catchError = (
```

Comment:
Could we consider moving this to elements to apply DRY principle?

Reply:
@phmngocnghia Yes can, but just considering where should we put it